src/gatekeeper.conf

```diff
@@ -0,0 +1,30 @@
+upstream-keepalives: false
+enable-encrypted-token: false
+enable-token-header: false
+enable-authorization-header: false
+enable-authorization-cookies: false
+enable-default-deny: false
+enable-refresh-tokens: true
+enable-logging: true
+preserve-host: true
+secure-cookie: true # SHOULD BE true!! Only set this to false for local dev
+server-write-timeout: 30000s # Timeout fetching complete response
+upstream-response-header-timeout: 30000s  # Time until header is received
```

Comment:
Should we remove these two timeout settings to use the default? 30000 is a bit much. (The default is 10s, that should be enough).

Reply:
Setup was copied from GOB-API, 10s should indeed be enough for GOB-StUf